### PR TITLE
[ci skip] Fix comment, since Rails 3.1 is out

### PR DESCRIPTION
**Pull request by claudiob**

### CODE REVIEW

**Comment by robin850** [comment]
Hi!

I guess we should avoid as much as possible speaking about versions. If you are reading the documentation, you should read it for your specific Rails version so we should just describe the available features. What do you tink ? :-)

File: activerecord/lib/active_record/migration.rb [hunk 1]
```diff
@@ -307,9 +307,9 @@ def initialize
   #
   # == Reversible Migrations
   #
-  # Starting with Rails 3.1, you will be able to define reversible migrations.
+  # Starting with Rails 3.1, you are able to define reversible migrations.
```
Comment (senny) [inline activerecord/lib/active_record/migration.rb]: Agree with @robin850. We can just say that this is possible, no need for the version back reference.

**Comment by claudiob** [comment]
@robin850 @senny We can go either way.

On one side, it's not a bad thing for people who started with old versions of Rails. For instance, say you are working on different Rails projects with different versions… it's helpful to know that something changed in Rails 3.1.

On the other side, a comment on top of the code might not be the "most appropriate" place where to find this reference, so we can remove it.

Let me know what you think. If you think removing is better, then I'll give it another pass to the whole codebase to find this sort of references.

**Comment by senny** [comment]
I'd remove the cross-reference.

While this can be an interesting information for a couple, such version cross-references could be spread all over the docs. It would be equally nice to know in the 3-2 docs that some pattern, method was replaced with rails 4. This is not something we can maintain in a sane fashion.

From my point of view, all the docs (guides, API) are version bound. They should describe that version and continue to be available when newer versions are released. The cross referencing can be done by the interested user.

**Comment by claudiob** [comment]
@senny Ready for you! :ribbon:

**Comment by senny** [comment]
@claudiob thank you :yellow_heart:

**Pull request merged**